demonstrations/tutorial_how_to_quantum_just_in_time_compile_vqe_catalyst.py

```diff
@@ -0,0 +1,200 @@
+r"""How to quantum just-in-time compile VQE with Catalyst
+=====================================================
+
+The `Variational Quantum Eigensolver <https://pennylane.ai/qml/demos/tutorial_vqe.html>`__ (VQE) is
+a widely used quantum algorithm with applications in quantum chemistry and portfolio optimization
+problems. It is an application of the `Ritz variational
+principle <https://en.wikipedia.org/wiki/Ritz_method>`__, where a quantum computer is trained to
+prepare the ground state of a given molecule.
```

Comment:
```suggestion
prepare the ground state of a given molecule. 
```

(empty commit to re-run)